documentation/how-to/digitize.en.md

```diff
@@ -139,6 +139,28 @@ A video demonstration of some editing possibilities:
 
 ![type:video](https://player.vimeo.com/video/499564854)<!-- markdown-link-check-disable-line -->
 
+## Erase Tool Functionality
+
+The erase tool is a new feature introduced in the geometry editor, designed to facilitate the removal of parts of a line or polygon geometry. This tool enables users to efficiently correct paths or polygons directly within the application interface and directly from the field.
+
+Suppose you have a polygon representing a building footprint, but there's an unwanted protrusion. You can use the erase tool to precisely remove that portion of the polygon, ensuring accuracy in your geometry representation.
+
+### How to Use
+
+1. **Select the Erase Tool**: Locate the erase tool icon in the toolbar of the geometry editor.
+
+2. **Choose Geometry to Erase**: Click on the line or polygon geometry that you want to edit using the erase tool.
+
+3. **Erase Segments or Portions**:
+
+    - For lines: Click and drag along the segment(s) you wish to erase.
+    - For polygons: Click within the area you want to erase, and drag to define the portion to be removed.
+4. **Confirm Erase Action**: Once you have selected the desired segments or portions to erase, release the mouse button to confirm the erase action.
+
+5. **Finalize Changes**: Save your changes or continue editing as needed.
```

Comment:
Wondering if we should get rid of this section. We do not go into that much detailing for other tools. I feel the two paragraphs introducing the feature + a screencast should be enough. We already go into more details on how to open the geometry editor toolbox to begin with.

Reply:
Yes, I think is better!